src/odemis/driver/picoquant.py

```diff
@@ -91,6 +167,17 @@ def __str__(self):
         return self.args[1]
 
 
+class HHError(Exception):
+    def __init__(self, errno, strerror, *args, **kwargs):
```

Comment:
Can you add a description when/why a HHError should be raised?

Reply:
Probably it is when the HydraHarp fails (because of the PR title) but that won't be in the file so just one sentence for what fail case this error is defined is enough I think.